### PR TITLE
Feat: remove command

### LINKED ISSUE #7: criar comando de remover do grupo (somente admin)

### PR DESCRIPTION
**Pull request by nickmaglowsch**
resolves: #7 

image
image

### CODE REVIEW

File: src/commands/remove.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+            const newUsers = group.users.filter((user) => !allUsers.find(u => _.isEqual(u, user)));
+
+            group.users = [ ...newUsers ];
+            logger.info("merge");
+            logger.info(group.users);
+
+            await group.save();
+
+            return "removido!";
```
Comment (alcuri) [inline src/commands/remove.ts]: Should be "Removido!", with first letter capitalized

File: src/mocks/factory.mock.ts [hunk 2]
```diff
@@ -4,6 +4,6 @@ import { TelegramFactory } from "../factory";
 export const baseFactory = (
     command: string,
     entities: TelegramBot.MessageEntity[] | undefined = undefined,
-    chatId: number = 123456,
+    chatId = 123456,
```
Comment (alcuri) [inline src/mocks/factory.mock.ts:7]: Why did you remove that types?

Comment (nickmaglowsch) [reply src/mocks/factory.mock.ts:7]: ESlint, if you are assigning a primitive type to a variable you should let it be inferred typed not implicated typed

**Review by alcuri — APPROVED** [review]
